inginious/frontend/static/css/INGInious.css

```diff
@@ -270,7 +285,3 @@ ul.checked-list-box input[type="checkbox"] {
   -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(102, 175, 233, 0.6);
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(102, 175, 233, 0.6);
 }
-.dragin{
```

Comment:
why removing this?

Reply:
It's what happens when you edit directly a generated file @Drumor 
https://github.com/UCL-INGI/INGInious/commit/59b490f3ea59c799ea72a88936fc2e1c64da0c24

The CSS files are generated from the LESS ones. You should not directly update the CSS, but rather the LESS file and run lessc to regenerate the css.

I'll add another commit to fix the bug ;-)